clouddriver-titus/clouddriver-titus.gradle

```diff
@@ -3,5 +3,9 @@ dependencies {
   compile project(":clouddriver-aws")
   compile spinnaker.dependency('bootActuator')
   compile spinnaker.dependency('bootWeb')
-  compile ('com.netflix:titan-client:0.28')
+  compile 'com.squareup.retrofit2:retrofit:2.0.0-beta4'
```

Comment:
we are going to land on these anyway from the azure client - used this change set to get a sense of how things look in retrofit2 land